### DISCUSSION TITLE
friends/

### DISCUSSION THREAD

**Discussion by giscus** [Yamabuki-bakery/tasty-discussion#4] [Announcements]
# friends/

以極限的風味挑起豐富你球網路文化世界的重任，，，

**Comment by PermeationLoop** [comment]
成爲我的 friend，做蕞幸福的壬，，，

請帶上妳的 URL，名稱，簡介和大頭貼，妳知道該怎麼找到我的🙂👍🏻

**Reply by PermeationLoop** [reply]
還有上表的群友看到這條請自覺聯繫我交換友鏈

**Reply by PermeationLoop** [reply]
附上與友鏈 SkyCanvas 的合影一張

<img src="https://github.com/user-attachments/assets/b477f6bf-a20e-4cdc-ae7d-0d9eec7d42d1" width="2560" height="1537" alt="SkyCanvas" />

**Comment by Parsifa1** [comment]
test333

**Reply by PermeationLoop** [reply]
捕捉
☝🏻️上面這位是🔴愉快的🟢辣妹羣友 云楼汐

**Comment by heyeahhhhhh** [comment]
上班时间偷偷视奸（

**Reply by PermeationLoop** [reply]
怎麼星期天下午還上班的

**Reply by heyeahhhhhh** [reply]
一个月放一天假喵😭

**Comment by erikopak-cmyk** [comment]
要足夠得聰明和邪惡，，，

**Comment by soildCoblat** [comment]
我呢我呢
<img src="https://imgs.qiubiaoqing.com/qiubiaoqing/imgs/680d1892786fce0t.jpeg" width="200">

**Reply by PermeationLoop** [reply]
妳是哪位啊（

**Reply by PermeationLoop** [reply]
<img width="256" height="252" alt="龍貓" src="https://github.com/user-attachments/assets/65418af7-7018-4380-b39c-c6d70dd31f14" />

**Comment by LingLambda** [comment]
好酷的blog主题

**Reply by PermeationLoop** [reply]
Ciallo！

感謝妳喜歡俺的主題，順便俺已順藤摸瓜通過您 GitHub 找到妳的 blog。
驚聞妳在手撕 Astro 主題，期待一下妳完工後的效果 😄

**Comment by LingLambda** [comment]
情报网吓晕了



---- 回复的原邮件 ----
| 发件人 | 開源 ***@***.***> |
| 日期 | 2026年03月28日 21:07 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.***>***@***.***> |
| 主题 | Re: [PermeationLoop/tasty-discussion] friends/ (Discussion #4) |

Ciallo！

感謝妳喜歡俺的主題，順便俺已順藤摸瓜通過您 GitHub 找到妳的 blog。
驚聞妳在手撕 Astro 主題，期待一下妳完工後的效果 😄

**Comment by LingLambda** [comment]
其实俺没打算优化blog主题了



---- 回复的原邮件 ----
| 发件人 | 開源 ***@***.***> |
| 日期 | 2026年03月28日 21:07 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.***>***@***.***> |
| 主题 | Re: [PermeationLoop/tasty-discussion] friends/ (Discussion #4) |

Ciallo！

感謝妳喜歡俺的主題，順便俺已順藤摸瓜通過您 GitHub 找到妳的 blog。
驚聞妳在手撕 Astro 主題，期待一下妳完工後的效果 😄

**Comment by LingXi9374** [comment]
看到今天中午那個Linux交流群出事了就又趕來了，由於事發後加不了你的好友，翻遍你的blog網站也沒發現其他的聯繫方式（比如郵箱），只好在這裡留言。很抱歉在這裡提一些不愉快的事情。還有但願那邊的相關人員不會過來看到這個評論））

怎麼說呢？我認為一個QQ群時有發生衝突是一件再正常不過的事，每個人都形形色色，在某些方面難免會有分歧，我非常理解你使用任何產品時對隱私的重視，也會有點同情，只是做法稍微過激了（每個人看到陰陽發言誰都會不舒服，是這樣的）。拋開影響來談，其實這種衝突只是小打小鬧罷了，但那邊的群主貌似心眼有點小（我待在那群很久了，群主什麼性格我比較清楚），直接踢人了事，換做我是群主我只會禁言幾分鐘給個冷靜期事後道歉讓這事過去就行了。

最後，我想告訴你的是不必把這事放在心上，繼續做好作為blogger的日常就行了。我本來未來計劃要寫幾篇博文，文章分別是關於 `搜索引擎安裝包投毒` 以及 `研究UnifiedPush消息推送` 兩個行文主題。本來計劃寫完直接在群裡at過來叫來看的，可惜現在只能換個方式保持聯繫力（悲）

如果覺得不方便，也歡迎來我的小群玩`549994140`，我也盡可能自己的努力把之前玩畫猜的群友也拉過來，這樣活動也能繼續進行（這天週六本來晚上你也要打算再次組織的，但是因為意外被干擾到了，我也理解，前面幾次沒參與是因為我上學了筆記本拿不到了）

**Reply by PermeationLoop** [reply]
感謝理解，有些話不便在這裏說，我已開通收信通道，煩請從頁面底部的 Email 入口給我發一條郵件，我會通過郵件回覆
